libs/as-sdk/assembly/tally.ts

```diff
@@ -24,7 +24,13 @@ export class RevealResult {
 }
 
 export default class Tally {
-	static getReveals(): RevealResult[] {
+	/**
+	 * Get the reveal data from the Oracle Program execution phase. This method also includes reveals which were not in
+	 * consensus. Use getReveals to only receive reveals which were in consensus.
+	 * @see {@link getReveals}
+	 * @returns All reveals from the Oracle Program execution phase.
+	 */
+	static getAllReveals(): RevealResult[] {
```

Comment:
Maybe we should call this something else? `getReveals` and `getAllReveals` are a lil too similiar (for the people who don't check docs). `getReveals_HAS_ALL_PLEASE_DONT_USE_OR_YOU_WILL_GET_FIRED`